integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -149,6 +149,37 @@ context(`Change links on the Check Your Answers page when updating an Induction`
       .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/qualification-details`)
       .submitPage()
 
+    // Change Worked before (Yes -> No)
```

Comment:
Similar cypress tests to above (the "create" journey), but this time for the "update" journey